src/vs/workbench/contrib/comments/browser/commentsViewActions.ts

```diff
@@ -168,3 +185,28 @@ registerAction2(class extends ViewAction<ICommentsView> {
 		view.filters.showResolved = !view.filters.showResolved;
 	}
 });
+
+registerAction2(class extends ViewAction<ICommentsView> {
+	constructor() {
+		super({
+			id: `workbench.actions.${COMMENTS_VIEW_ID}.toggleSortByUpdatedAt`,
+			title: localize('toggle sorting by updated at', "Sort by Updated At"),
+			category: localize('comments', "Comments"),
+			toggled: {
+				condition: CONTEXT_KEY_SORT_BY_UPDATED_AT,
```

Comment:
Instead of creating the expression above, could you inline it here? 

```suggestion
				condition: ContextKeyExpr.equals('commentsView.sortBy', 'updatedAt'),
```